core/src/main/java/io/jenkins/pluginhealth/scoring/scores/AdoptionScoring.java

```diff
@@ -126,73 +106,23 @@ public ScoringComponentResult getScore(Plugin plugin, Map<String, ProbeResult> p
-                @Override
-                public String getDescription() {
-                    return "There must be a commit between 2 years and 4 years since the last release.";
-                }
-
-                @Override
-                public ScoringComponentResult getScore(Plugin plugin, Map<String, ProbeResult> probeResults) {
-                    final ProbeResult probeResult = probeResults.get(LastCommitDateProbe.KEY);
-                    if (probeResult == null || ProbeResult.Status.ERROR.equals(probeResult.status())) {
-                        return new ScoringComponentResult(-100, 100, List.of("Cannot determine the last commit date."));
+                    if (days < Duration.of(30 * 12 + 1, ChronoUnit.DAYS).toDays()) {
+                        return new ScoringComponentResult(60, getWeight(), List.of(defaultReason, "Less than a year between last release and last commit."));
                     }
-                    final long days = getTimeBetweenLastCommitAndDate(probeResult.message(), plugin.getReleaseTimestamp().withZoneSameInstant(getZone())).toDays();
-                    if (days <= Duration.of(4 * 365, ChronoUnit.DAYS).toDays()) {
-                        return new ScoringComponentResult(100, getWeight(), List.of("There is less than 4 years between the last commit and the last release."));
+                    if (days < Duration.of(30 * 24 + 1, ChronoUnit.DAYS).toDays()) {
+                        return new ScoringComponentResult(20, getWeight(), List.of(defaultReason, "Less than 2 years between last release and last commit."));
                     }
-                    return new ScoringComponentResult(0, getWeight(), List.of("No commit in the last 4 years."));
-                }
-            },
-            new TimeSinceLastCommitScoringComponent() {
-                @Override
-                public String getDescription() {
-                    return "It must have less than 4 years between the last commit and the last release.";
-                }
-
-                @Override
-                public ScoringComponentResult getScore(Plugin plugin, Map<String, ProbeResult> probeResults) {
-                    final ProbeResult probeResult = probeResults.get(LastCommitDateProbe.KEY);
-                    if (probeResult == null || ProbeResult.Status.ERROR.equals(probeResult.status())) {
-                        return new ScoringComponentResult(-100, 100, List.of("Cannot determine the last commit date."));
+                    if (days < Duration.of(30 * 48 + 1, ChronoUnit.DAYS).toDays()) {
+                        return new ScoringComponentResult(10, 2, List.of(defaultReason, "Less than 4 years between last release and last commit."));
                     }
-                    final long days = getTimeBetweenLastCommitAndDate(probeResult.message(), plugin.getReleaseTimestamp().withZoneSameInstant(getZone())).toDays();
                     if (days > Duration.of(4 * 365, ChronoUnit.DAYS).toDays()) {
```

Comment:
The check right above is for `days <  30*48+1`, so checking `days > 4 * 365` will return almost always true (unless you hit the interval between 1441 and 1460 days caused by discrepancy between the time conversions).

Reply:
I changed that in https://github.com/jenkins-infra/plugin-health-scoring/pull/392/commits/b8b1238867ef8e5f2e4879b975a79f56d9ea5259